tasklist/hi.json

```
@@ -0,0 +1,520 @@
+    " TELEMETRY_INTRODUCTION_EXPLANATION_2 " : " । इसलिए हम स्वैच्छिक टेलीमेटरी सुविधाओं शुरू कर रहे हैं, तो हम समझ सकते हैं जो इंजन की जरूरत में सुधार लाने और कौन सी सुविधाएं हम आपको अगले लाना चाहिए के कुछ हिस्सों " ,
+    " TELEMETRY_INTRODUCTION_EXPLANATION_3 " : " आप व्यापक समुदाय और मदद हमें बीपीएम इंजन में सुधार करने के लिए योगदान करना चाहते हैं, टेलीमेटरी को सक्षम करने पर विचार करें। " ,
+    " TELEMETRY_INTRODUCTION_COMMUNITY " : " व्यापक समुदाय में योगदान करें " ,
+    " TELEMETRY_INTRODUCTION_IMPROVE " : " कैमुंडा बीपीएम इंजन को बेहतर बनाने में मदद करें " ,
+    " TELEMETRY_DETAILS_HEADER " : " क्या टेलीमेटरी मापता है? " ,
+    " TELEMETRY_DETAILS_META_DATA " : " मेटा डेटा " ,
+    " TELEMETRY_DETAILS_META_DATA_WHAT " : " उत्पाद संस्करण, संस्करण, डेटाबेस तकनीक, JDK विक्रेता और संस्करण, एप्लिकेशन सर्वर विक्रेता और संस्करण, लाइसेंस कुंजी जानकारी यदि प्रदान की गई है (ग्राहक का नाम, समाप्ति तिथि, सक्षम सुविधाएँ), और छद्म नामांकित संस्थापन ID। " , " ।
+    " TELEMETRY_DETAILS_META_DATA_WHY " : " । यह हमें बेहतर वातावरण जहां Camunda चल रहा है को समझने के लिए " ,
+    " TELEMETRY_DETAILS_USAGE_STATISTICS " : " उपयोग के आंकड़े " ,
+    " TELEMETRY_DETAILS_USAGE_STATISTICS_WHAT " : " निष्पादित API आदेशों की गणना, निष्पादन मेट्रिक्स की गणना (मूल प्रक्रिया इंस्टेंसेस, गतिविधि उदाहरण शुरू करना, निर्णय उदाहरण निष्पादित करना, निर्णय तत्वों को निष्पादित करना, अद्वितीय कार्य कर्मी), और कैमुंडा एकीकरण प्रकार (उदाहरण के लिए, चाहे स्प्रिंग बूट स्टार्टर, कैमुंडा) बीपीएम भागो, Wildfly एकीकरण या EJB एकीकरण उपयोग किया जाता है)। " ,
+    " TELEMETRY_PREFERENCES_HEADER " : " गोपनीयता प्राथमिकताएं " ,
+    " TELEMETRY_PREFERENCES_DETAILS " : " ।। बढ़ाने के लिए उपयोगकर्ता अनुभव, Camunda बीपीएम रनटाइम इंजन Camunda सेवाएं जीएमबीएच, जो बाहरी नेटवर्क अनुरोधों की आवश्यकता है के साथ एकीकृत कर सकते हैं कृपया नीचे सेटिंग से चुनने " ,
+    " TELEMETRY_PREFERENCES_META " : " मेटा और उपयोग सांख्यिकी सक्षम करें " ,
+    " TELEMETRY_PREFERENCES_META_DETAILS " : " Camunda बीपीएम रनटाइम उपयोग के आंकड़े भेजने की अनुमति दें हम इस जानकारी का उपयोग विशेष रूप से आप पर्यावरण प्रयोग कर रहे हैं में एक स्थिर और बेहतर उत्पाद अनुभव प्रदान करने के लिए इस उत्पाद संस्करण के बारे में कलेक्ट जानकारी, तकनीकी वातावरण और कैसे Camunda अनुमति देता है।। सुविधाओं उपयोग में हैं। यह जानकारी आम तौर पर तकनीकी और एकीकृत की है, लेकिन सीमित है और व्यक्तिगत डेटा pseudonymized शामिल हो सकते हैं। " ,
+    " TELEMETRY_PREFERENCES_DISCLAIMER " : " इन विकल्पों में, आपकी व्यक्तिगत जानकारी में से कोई भी प्रस्तुत किया जाएगा के किसी भी है। " ,
+    " TELEMETRY_FOOTER_LEARN_MORE " : "और जानें: " ,
+    " TELEMETRY_FOOTER_PRIVACY_POLICY " : " कैमुंडा गोपनीयता नीति " ,
+    " TELEMETRY_ERROR_STATUS " : " त्रुटि " ,
+    " TELEMETRY_ERROR_MESSAGE " : " आपका कॉन्फ़िगरेशन सहेजा नहीं जा सका। कृपया पुनः प्रयास करें। "
+  },
+  " डेटक्लोलेस " : {
+    " महीने " : [
```

Same as with the 'labels' property, don't translate the property names

```suggestion
  "dateLocales": {
    "months": [
```